README.md

```diff
@@ -1,2 +1,103 @@
 # Hello-Microverse
-First Microverse Project on Day 2
+
+[Hello-Microverse] is an open source project to familiarize new Micronauts with the Microverse Programme.
+
+## 🛠 Built With
+Client: [React], [Redux]
+Server: [Node.js]
+Database [MongoDB]
+
+## Key Features
+[Hello-Microverse] key features are:
+
+[create-a-new-project-from-templates]
+[Add-new-collaborators]
+[Add-new-templates-for-projects]
+(back to top)
+
+## Live Demo link
+[Hello-Microverse](https://github.com/diarisdiakite/Hello-Microverse/pull/1)
+(back to top)
```

Comment:
- [ ] [OPTIONAL] Please I suggest you remove the live demo link since this project is not deployed